src/borrow_tracker/tree_borrows/tree.rs

```diff
@@ -658,25 +670,26 @@ impl AccessRelatedness {
     /// for the child node. This function assumes that we propagate away from the initial
     /// access.
     pub fn for_child(self) -> Self {
-        use AccessRelatedness::*;
+        use AccessRelatedness as Rel;
```

Comment:
Why the renames? I find that more confusing than `::*` because it means one has to learn a new name.